oauth2/handler.go

```diff
@@ -1154,3 +1207,175 @@ func (h *Handler) logOrAudit(err error, r *http.Request) {
 		x.LogAudit(r, err, h.r.Logger())
 	}
 }
+
+// swagger:route POST /credentials oAuth2 createVerifiableCredential
```

Comment:
Consider moving this to the oidc tag (i believe we have such a tag if i am not mistaken)